### ISSUE TITLE
drawn mask: cannot list shapes used

### ISSUE THREAD

**Issue by darkbasic** [darktable-org/darktable#12986]
**Describe the bug/issue**
Whenever you create a drawn mask from any module (in this example filmic rgb) if you click on the small arrow near the used shapes which is supposed to list them it doesn't work.

**To Reproduce**
1. Go to 'filmic rgb' module
2. Click on 'drawn mask'
3. Add some shapes
4. Click on the little arrow near the used shapes

**Expected behavior**
It should list the used shapes.

**Screencast**
https://www.youtube.com/watch?v=ENbDJjphZtY

**Which commit introduced the error**
I have no idea if it used to work or not.

**Platform**

* darktable version : 4.1.0+1086~ge56f3d964-dirty
* OS                : Linux - kernel 6.0.9
* Linux - Distro    : Gentoo ppc64le (4K page size)
* Memory            : 64 GB
* Graphics card     : AMD RX 570
* Graphics driver   : mesa git karolherbst/mesa:rusticl/si
* OpenCL installed  : rusticl
* OpenCL activated  : no
* Xorg              : Wayland
* Desktop           : Gnome 43
* GTK+              : 3.24.34
* gcc               : 11.3.0
* cflags            : -O2 -pipe -mcpu=power9 -mtune=power9
* CMAKE_BUILD_TYPE  : RELWITHDEBINFO

**Additional context**

**Comment by dterrahe** [comment]
That drop down would list the shapes that exist, for example in _other_ modules, that you could include in the mask for the current one. The shapes that are _already_ used in the current module can be shown by clicking the right-most button in the row of shapes.

I don't know how well this is explained in the manual. You could suggest improvements there or improvements here that make the functionality more obvious.

**Comment by darkbasic** [comment]
You're right, maybe we could gray out the small arrow whenever there are no available shapes and so there is no drop down menu to show?

**Comment by dterrahe** [comment]
Sounds like a good idea; please feel free to submit a pr.
